nise/report.py

```diff
@@ -771,6 +773,16 @@ def ocp_create_report(options):  # noqa: C901
     end_date = options.get("end_date")
     cluster_id = options.get("ocp_cluster_id")
     static_report_data = options.get("static_report_data")
+    if "ros_ocp_info" in options:
+        if options.get("ros_ocp_info") in ["Yes", "yes"]:
+            ros_ocp_info = True
+        else:
+            LOG.warning("--ros_ocp_info parameter only accepting Yes/yes value to create ROS-openshift. If we pass other than that it will ignore data creation for ROS-Openshift.")
+            ros_ocp_info = False
+    else:
+        ros_ocp_info = False
```

Comment:
If you change the CLI flag as suggested, all of this can be replaced with:
```suggestion
    ros_ocp_info = options.get("ros_ocp_info")
```